tests/settings/settings_wallet/test_wallet_settings_watched_addr_include_in_total_balance.py

```diff
@@ -23,6 +23,7 @@
     pytest.param('0x7f1502605A2f2Cc01f9f4E7dd55e549954A8cD0C', ''.join(random.choices(string.ascii_letters +
                                                                                       string.digits, k=20)))
 ])
+@pytest.mark.skip(reason="https://github.com/status-im/status-desktop/issues/14862")
```

Comment:
Can we please add also this reference? `https://github.com/status-im/status-desktop/issues/14509` sorry about that , i forgot to tell you at the very beginning

Reply:
Done